test/bootloader_proposal_client_test.rb

```diff
@@ -222,5 +222,14 @@
       expect(Yast::PackagesProposal).to receive(:AddResolvables).with("yast2-bootloader", :package, packages_to_propose)
       subject.make_proposal({})
     end
+
+    it "returns warning if old system use different boot technology then new one" do
+      Yast.import "Mode"
```

Comment:
I'd move the import to the top of the file (and remove it from the other places), it will stay imported for the other tests anyway.